kubes-pod.yaml

```diff
@@ -0,0 +1,56 @@
+spec:
+  containers:
+  - env:
+    - name: "DOCKER_TLS_CERTDIR"
+      value: "/certs"
+    image: "docker:20.10-dind"
+    imagePullPolicy: "Always"
+    name: "dind"
+    securityContext:
+      privileged: true
+      runAsUser: 0
+    volumeMounts:
+    - mountPath: "/certs/client"
+      name: "docker-secrets"
+      readOnly: false
+      #    - mountPath: "/var/lib/docker"
+      #      name: "docker-cache"
+      #      readOnly: false
+  - env:
+    - name: "DOCKER_TLS_VERIFY"
+      value: "1"
+    - name: "DOCKER_HOST"
+      value: "tcp://localhost:2376"
+    - name: "DOCKER_CERT_PATH"
+      value: "/certs/client"
+    - name: "_JAVA_OPTIONS"
+      value: "-Duser.home=/home/jenkins"
+    - name: "JDKW_VERBOSE"
+      value: "1"
+    - name: "HOME"
+      value: "/home/jenkins"
+    image: "docker.arpnetworking.com/build-base"
```

Comment:
Is this right for ISM?

Reply:
This is right. This is the docker container in the build system that is used as the base